pyproject.toml

```diff
@@ -164,7 +164,7 @@ typing-extensions = ">=3.10.0.1"
 cryptography = ">=3.4.7"
 # ijson 3.1.4 fixes a bug with "." in property names
 ijson = ">=3.1.4"
-matrix-common = "^1.2.1"
+matrix-common = "^1.3.0"
```

Comment:
I'm not sure if we need to confer with packagers/wait a little while before bumping this dependency in Synapse?